### PR TITLE
Post for the 2.1.0 and 0.7.0 releases

### PR DESCRIPTION
**Pull request by parkouss**

### CODE REVIEW

**Comment by parkouss** [comment]
@wlach Do you mind reviewing this ?

Thanks :)

File: _posts/2015-11-24-2.1.0-release-and-gui-0.7.0.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+- allow to run a single build - equivalent to **-\-launch** argument ([bug 1231399])
+- fixed a regression from 0.6.0 that prevented to show the last step
+  of the bisection ([bug 1232261])
+- added profile persistence support ([bug 1220553])
+
+And other fixes: [bug 1232660] and [bug 1233657]
+
+Thanks to [Saurabh Singhal] and [GopianiS]! They are new mozregression
+contributors and they helped me for this release.
```
Comment (wlach) [inline _posts/2015-11-24-2.1.0-release-and-gui-0.7.0.md]: and they helped me for this release -> who helped me with this release

File: _posts/2015-11-24-2.1.0-release-and-gui-0.7.0.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+---
+title:  "Release 2.1.0 and GUI release 0.7.0"
+author: Julien Pagès
+---
+
+Minor releases of the command line and GUI mozregression tools.
```
Comment (wlach) [inline _posts/2015-11-24-2.1.0-release-and-gui-0.7.0.md]: Minor releases of -> This is a minor release of
command line -> command-line

File: _posts/2015-11-24-2.1.0-release-and-gui-0.7.0.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+On the command line side:
+
+- Added basic support for jsshell integration ([bug 1232036])
+- Reduced build launch time ([bug 1216232])
+- Added certificate based TaskCluster authentication ([bug 1227780])
```
Comment (wlach) [inline _posts/2015-11-24-2.1.0-release-and-gui-0.7.0.md]: certificate based -> certificate-based

**Comment by wlach** [comment]
This looks great, some minor suggestions inline.

**Pull request merged**